fw/connection.h

```diff
@@ -404,6 +404,8 @@ tfw_connection_put(TfwConn *conn)
 		return;
 
 	rc = atomic_dec_return(&conn->refcnt);
+	BUG_ON(rc < 0 && rc != TFW_CONN_DEATHCNT);
```

Comment:
`TFW_CONN_DEATHCNT` is definitely greater than zero. What exactly is this `BUG_ON()` for?

Reply:
#define TFW_CONN_DEATHCNT	(INT_MIN / 2) is less than zero, but BUG_ON should be BUG_ON(rc < TFW_CONN_DEATHCNT)

Reply:
Ah, indeed. Thanks!